### DISCUSSION TITLE
Getting started...

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#5] [Announcements]
## Getting started with Speedtest Tracker

Should you have any questions first check out the [README](https://github.com/alexjustesen/speedtest-tracker#readme) and the [docs](https://docs.speedtest-tracker.dev/), if you still have a question please start a discussion topic.

**Comment by Bulletoverload** [comment]
I appreciate the work you are doing here. Really nice to have some updates to the old speedtest tracker, excited to see where you take it. Thanks!

Also, not sure if you saw this/this was you, but there is a forum post on unraid.net you may want to be involved with.

https://forums.unraid.net/topic/130245-support-devzwf-speedtest-tracker/#comment-1185366

**Reply by alexjustesen** [reply]
That was not me, and thanks for the heads up! Oh and also for supporting the project 🍻

**Comment by Bulletoverload** [comment]
I figured it wasn’t you. That forum is a great place for rallying and giving support and interacting with the unraid community. Happy to support and help in any way I can!

**Reply by alexjustesen** [reply]
Ya know I've had an unraid server running for 8 years almost and I always forget about the forums...
